meal.go

```diff
@@ -93,21 +94,25 @@ func GetMealIds(c *gin.Context) {
 	})
 }
 
+// 실제 식단 데이터
 type Diet struct {
 	Diet    string `json:"diet"`
 	Calorie string `json:"calorie"`
 }
 
+// 점심 학식
```

Comment:
Codacy found an issue: [comment on exported type Lunch should be of the form "Lunch ..." (with optional leading article)](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091160)